### PR TITLE
Gg/improve opt mainnet deployment

### PR DESCRIPTION
**Pull request by peculiarity**

### CODE REVIEW

File: scripts/deployment20/deploy.ts [hunk 1]
```diff
@@ -125,6 +125,7 @@ export class DeploymentSystem extends DeployedSystemHelpers {
     } else {
       // if forked other network then merge configs files
       if (this.forkedNetwork) {
+        console.log('Am I here?')
```
Comment (zerotucks) [inline scripts/deployment20/deploy.ts]:
```suggestion
```

File: scripts/deployment20/deploy.ts [hunk 2]
```diff
@@ -446,7 +458,7 @@ export class DeploymentSystem extends DeployedSystemHelpers {
     !useInch &&
       addLocalEntries &&
       (await deploySwapContract.setPool(
-        this.config.common.STETH.address,
+        this.config.common.WSTETH.address,
```
Comment (zerotucks) [inline scripts/deployment20/deploy.ts]:
```suggestion
        this.config.common.STETH.address,
```

Changing that will possibly mess with mainnet uniswap tests. Any STETH positions

File: scripts/deployment20/deploy.ts [hunk 3]
```diff
@@ -463,19 +475,15 @@ export class DeploymentSystem extends DeployedSystemHelpers {
...
     )
 
-    const operationsRegistry: OperationsRegistry = new OperationsRegistry(
-      this.deployedSystem.OperationsRegistry.contract.address,
-      this.signer,
-    )
     const dsProxyRegistry = await this.ethers.getContractAt(
       DS_PROXY_REGISTRY_ABI,
-      this.config.common.DsProxyRegistry.address,
+      this.config.common.DS_PROXY_REGISTRY.address,
```
Comment (zerotucks) [inline scripts/deployment20/deploy.ts]: This doesn't look right to me

File: scripts/deployment20/deploy.ts [hunk 4]
```diff
@@ -463,19 +475,15 @@ export class DeploymentSystem extends DeployedSystemHelpers {
...
       this.signer,
     )
 
     this.deployedSystem['DsProxyRegistry'] = { contract: dsProxyRegistry, config: {}, hash: '' }
 
-    await this.addAllEntries(operationsRegistry)
+    await this.addAllEntries()
```
Comment (zerotucks) [inline scripts/deployment20/deploy.ts:485]: Cool

File: test/e2e/strategies/aave/Open.test.ts [hunk 5]
```diff
@@ -318,13 +314,12 @@ describe(`Strategy | AAVE | Open Position`, async function () {
...
-            use1inch: true,
-            network: networkFork,
-            systemConfigPath: `./test-configs/${networkFork}.conf.ts`,
-          }),
-        )
+        fixture = await getSystemWithAaveV3Positions({
+          use1inch: true,
+          network: networkFork,
+          systemConfigPath: `./test-configs/${networkFork}.conf.ts`,
+          configExtentionPaths: [`./test-configs/swap.conf.ts`],
```
Comment (zerotucks) [inline test/e2e/strategies/aave/Open.test.ts:321]: nice. I'll want to merge this change as think it's better than what I've ended up doing

**Review by zerotucks — APPROVED** [review]

File: scripts/deployment20/deploy.ts (hunk 2 above)
Comment (peculiarity) [reply scripts/deployment20/deploy.ts]: This should be reverts as I don't use setupLocalSystem at all

**Pull request merged**